### PR TITLE
Custom items, second wave.

### PR DESCRIPTION
**Pull request by Alberyk**
Contains the following items:

Borderworlds Turtleneck - Jaylor Rameau - evilbrage
Consecrated Athame - Tina Kaekel - tainavaa
Hephaestus Industrial Exosuit MK III Customization Kit - Zairjah - alberyk
Inscribed Silver-handled Cane - Usiki Guwan - fireandglory
Black Left Glove - Kathleen Bullard - valky_walky2
Sobriety Chip - Yoiko Ali - raineko
Loyalty Implant - Eric Derringer - xelnagahunter
Loyalty Implant - Clement Beach - icuris
Unathi Wrappings - Azeazekal Karnaikai - canon35
Unathi head wrappings - Azeazekal Karnaikai - canon35
ATLAS poster - Conservan Xullie - conservatron
Fire Extinguisher Zippo - Jacob Locke - completegarbage
MUM AI core - MUM - bygonehero
Sketchpad - Zakiya Ahmad - sierrakomodo
Sketching pencil - Zakiya Ahmad - sierrakomodo
Broken Vaurca Mandible - Ka'Akaix'Zah Void - sleepywolf
Shaman Hoodie - Fereydoun Nioathi - jackboot

Updated item config: https://www.dropbox.com/s/9zi9t9zrcuqfrvg/custom_items.txt?dl=0
Updated Synth custom sprite config: https://www.dropbox.com/s/ifnx0lhihgpoian/custom_sprites.txt?dl=0

### CODE REVIEW

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -144,3 +144,215 @@
...
+
+/obj/item/weapon/fluff/derringer_implanter //Loyalty Implant Activator - Eric Derringer - xelnagahunter - DONE
+	name = "loyalty implanter activator"
+	desc = "A small device used to activate a loyalty implant of a certain owner. This one has a tag: 'Eric Derringer'."
+	icon = 'icons/obj/syndieweapons.dmi'
+	icon_state = "OLDc-4detonator_1"
+	w_class = 1.0
+	var/obj/item/weapon/implant/imp = null
+
+/obj/item/weapon/fluff/derringer_implanter/attack_self(mob/user as mob)
```
Comment (skull132) [inline code/modules/customitems/item_defines.dm]: Can you not generalize this?

```
/obj/item/weapon/implanter/fluff
	var/allowed_ckey = ""
	var/implant_type = null

/obj/item/weapon/implanter/fluff/proc/create_implant()
	if (!implant_type)
		return
	imp = new implant_type(src)
	update()

	return

/obj/item/weapon/implanter/fluff/attack(mob/M as mob, mob/user as mob)
	if (!M.ckey || M.ckey != allowed_ckey)
		return

	..()
```

And into item_spawning.dm:
```
	else if (istype(item, /obj/item/weapon/implanter/fluff))
		var/obj/item/weapon/implanter/fluff/L = item
		L.allowed_ckey = assoc_ckey
		L.implant_type = text2path(additional_data)
```

Comment (Alberyk) [inline code/modules/customitems/item_defines.dm]: Change done.

File: code/modules/customitems/item_spawning.dm [hunk 2]
```diff
@@ -80,6 +80,13 @@
 			var/obj/item/device/kit/suit/kit = item
 			kit.new_light_overlay = additional_data
 			kit.new_mob_icon_file = CUSTOM_ITEM_MOB
+			
+	// for snowflake implants
+	if(istype(item, /obj/item/weapon/implanter/fluff))
```
Comment (skull132) [inline code/modules/customitems/item_spawning.dm]: This should be an `else if`, since only one condition can be true.

Comment (Alberyk) [inline code/modules/customitems/item_spawning.dm]: Done.

**Comment by NanakoAC** [comment]
exosuit customisation kit? please elaborate

**Comment by Alberyk** [comment]
It changes the sprite, name and description of a certain exosuit when you apply the item to it.

**Pull request merged**